.cspell.json

```
@@ -803,6 +804,7 @@
     "apps/api/src/app/workflows-v2/maily-test-data.ts",
     "apps/api/src/app/workflows-v2/usecases/validate-content/validate-placeholders/validate-placeholder.usecase.ts",
     "apps/dashboard/eslint.config.js",
+    "packages/internal-sdk/**/*",
```

As discussed, this will leave under libs since it's an internal SDK.
```suggestion
    "libs/internal-sdk/**/*",
```